content/_index.md

```diff
@@ -99,6 +99,22 @@ services - be that for instant messages, VoIP call setups, or any other
 objects that need to be reliably and persistently pushed from A to B in
 an interoperable and federated manner.
 
+### Bindingness of requirements
+
+As a standards document, the Matrix Spec needs to signify the bindingness of the
+individual features, APIs, procedures, or other requirements it describes.
```

Comment:
"Bindingness" is rather an awkward word!

Can we just say something like:

```suggestion
### Use of RFC2119 keywords
```

... and omit the first paragraph, which I don't think adds much value?

Reply:
I tried to be more generic and may have ended up with a mangled Germenglish version. How do you think about something slightly more general and introductory than jumping straight to RFC 2119 in the title?

```suggestion
### Requirement levels
```

---

I'm slightly confused by the spelling in your proposal: Is it correct that contrary to MSCs, RFCs use a space before the number? RFC 2119 does where it references itself.

Reply:
> I'm slightly confused by the spelling in your proposal: Is it correct that contrary to MSCs, RFCs use a space before the number?

No, that's just me being a crank. The space is correct.